docs/src/main/asciidoc/datasource-guide.adoc

```diff
@@ -230,5 +230,12 @@ AgroalDataSource dataSource2;
 [[configuration-reference]]
 == Agroal Configuration Reference
 
+|===
```

Comment:
This doc change is not needed since the config documentation will be generated automatically from the javadoc: (thanks to the include directive below)

Reply:
Thanks for notifying me that. I'll change it.

Reply:
Yup and CI is failing because of this so better get rid of it!